src/main/java/org/apache/commons/net/nntp/NNTPClient.java

```diff
@@ -1283,7 +1283,7 @@ public boolean selectArticle(final long articleNumber, final ArticleInfo pointer
     /**
      * Same as <code> selectArticle(articleId, (ArticleInfo) null) </code>
      *
-     * @param articleId the article Id
+     * @param articleId the article's identifier
```

Comment:
Superfluous change IMO, this change makes the text wordier, it was clearer to me previously.